tests/unit/flow/test_flow_multimode.py

```diff
@@ -79,11 +79,13 @@ def input_fn():
                                                   [1.0, 1.0, 1.0]]))
 
     chunkIndexer1 = BinaryPbIndexer.load('kvidx1.bin')
-    assert chunkIndexer1.size == 6
+    assert chunkIndexer1.size == 3
```

Comment:
Why does the `chunkIndexer` size become half now?

Reply:
Before, the indexer indexed all documents twice, since the `recur_on` was not set to `chunks` for this indexer. So the real question is: why was it 6 beforehand ;)